### PR TITLE
[IOTDB-2937] Add influxdb verificationQuery for iotdb-benchmark

### PR DESCRIPTION
**Pull request by jun0315**
see https://issues.apache.org/jira/browse/IOTDB-2937

### CODE REVIEW

**Comment by jun0315** [comment]
PTAL~ @SteveYurongSu @SpriCoder

File: README.md [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ INGESTION           0.39        0.10        0.12        0.13        0.15
 
 Before running this mode, you need to use the correctness write mode to write data to the database.
 
-In order to verify the correctness of the data set writing, you can use this mode to query the data set written to the database. Currently this mode only supports IoTDB v0.12 and v0.13
+In order to verify the correctness of the data set writing, you can use this mode to query the data set written to the database. Currently this mode only supports IoTDB v0.12 , v0.13 and influxdb v1.x
```
Comment (SpriCoder) [inline README.md:1046]: Please update README-cn.md at the same time.

**Review by SpriCoder — CHANGES_REQUESTED** [review]
Excellent job! Only a minor problem need to be fixed.

File: README.md (hunk 1 above)
Comment (jun0315) [reply README.md:1046]: Done~

**Review by SpriCoder — APPROVED** [review]
LGTM!

**Pull request merged**